docs/tutorial/visualization_widget.rst

```diff
@@ -0,0 +1,17 @@
+3D Visualization Widget
+------------------------
+
+If you want to quickly investigate many mappings in 3D, we recommend using
+our Jupyter Notebook widget, which comes with a slider and an interactive 3D
+visualization::
+
+    from kartograf.utils.mapping_visualization_widget import display_mappings_3d
+
+    # here a list of three mappings
+    mappings: List[AtomMappings] = [mappingA, mappingB, mappingZ] #pseudo code
```

Comment:
could we build in some dummy mappings into the package that would help with demonstrations like this?

`from kartograf.examples import demo_mappings` or something?

Reply:
that is a good Idea!

Reply:
will add this one to the project board.